types/locomotive-scroll/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": ["es6", "dom"],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": ["../"],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true
```

Comment:
Do you need to enable this option? We usually don't want DT packages to enable extra options, and leave things like `esModuleInterop` to be used by consumers of the DT definitions.

Reply:
If I don't set `esModuleInterop` to `true`, then in my test file I get this error in VSCode: *This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.* and the typing doesn't work.

Also I get *SyntaxError: Unexpected token } in JSON at position 390* when running `npm test locomotive-scroll` with `esModuleInterope` set to `false`.

I might just go back to where I was before the support for CommonJS. What do you think is the best ?